armada/handlers/k8s.py

```diff
@@ -58,3 +58,10 @@ def get_namespace_pod(self, namespace="default"):
         '''
 
         return self.client.list_namespaced_pod(namespace)
+
+    def get_all_pods(self):
+        '''
+        This will return a list of pods across all namespaces
+        '''
+
+        return self.client.list_pod_for_all_namespaces()
```

Comment:
I agree with that thinking. Along that same line, the k8s class doesn't seem very necessary since most of its functionality is just calling the kubernetes api. I was just following the structure that seemed to already be in place